src/chi-vue/src/components/data-table/data-table.scss

```diff
@@ -37,6 +37,14 @@
   text-overflow: ellipsis;
   white-space: nowrap;
 
+  .-cell-wrap {
```

Comment:
Include a **TODO** comment here saying to remove it once a new version of Chi is released.

Reply:
Done

Reply:
Done!